shell/ci/release/docker.sh

```diff
@@ -173,7 +173,13 @@ build_and_push_image() {
   fi
   (
     set -x
```

Comment:
Can you remove the `set -x`, once there's `if` statements in here, it just makes it super complicated to read the output 😓 

```suggestion
```